lib/terraform/runner/request.rb

```diff
@@ -24,10 +24,13 @@ def initialize(action_type)
 
       def build_json_post_arguments
         case @action_type
-        when ActionType::CREATE, ActionType::APPLY, ActionType::DELETE
-          if !@request_arguments.key?(:cloud_providers)
-            @request_arguments[:cloud_providers] = []
-          end
+        when ActionType::CREATE
+          @request_arguments[:name] = random_stack_name if !@request_arguments.key?(:name)
```

Comment:
Prefer `unless @request_arguments.key?(:name)` instead of `if !` for simple single arg conditionals like this

Reply:
Unless `@request_arguments[:name] = nil` is a valid case you want to allow, you could simplify this as
```suggestion
          @request_arguments[:name] ||= random_stack_name
```